src/components/NavSection.vue

```diff
@@ -51,18 +61,21 @@
           </div>
         </form>
       </div>
+
       <div class="navbar-end">
         <Dropdown v-slot="{ onFocus }" :text="$t('header.about-tab')">
           <NuxtLink
             class="navbar-item"
             :to="localePath('/about')"
+            role="menuitem"
```

Comment:
Nice catch, but instead of duplicating this (and requiring that the `Dropdown` consumer know to repeat it everywhere) could we just add an `a11yProps` to the slot that includes the role and then `v-bind` that to all the dropdown items?

Reply:
Makes sense, implemented in c7231c1.